### ISSUE TITLE
Different translation for same string

### ISSUE THREAD

**Issue by basile-laderchi** [bitjson/gulp-l10n#31]
I have the following scenario:
I have a table with the header cells displaying only the first letter of the day: "M T W T F S S", but in Greek that I am trying to translate the two "T"s should be translated differently. The same goes for the two "S"s.
Is there any way I could distinguish between those two?

Maybe when generating the hash and checking back an attribute could be hashed along with the text?

**Comment by bitjson** [comment]
@basile-laderchi would a solution [like this](http://stackoverflow.com/a/18587634) work for you? You could translate the full name (which is the more logical thing to translate anyways), and then only display the first letter. Also means your locale file won't be full of hard-to-understand values.

**Comment by basile-laderchi** [comment]
@bitjson to be completely honest the localized content is part of an inline svg.
The localization works great (only thing I did is to add the "tspan" to the elements to be exported).

Unfortunately after several tests and searching on the net I found out that the ":first-letter" pseudo selector doesn't work with svg, but having jquery already loaded I re-set their content only to the first letter of their content.
